### PR TITLE
Added basic motor board support

### PR DESCRIPTION
**Pull request by vlermuis**
Updated cmake file

### CODE REVIEW

File: rasp_tcp_srv/src/motor_ctrl.c [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+void update_speed()
+{
+	softPwmWrite (motor1_pwm_pin, current_right_speed);
+	softPwmWrite (motor2_pwm_pin, current_left_speed);
+}
+
+int forward()
+{
+    current_right_speed += SPEED_STEP;
```
Comment (Vulbado) [inline rasp_tcp_srv/src/motor_ctrl.c:64]: Any limitation for MAX speed?

File: rasp_tcp_srv/src/rasp_srv.c [hunk 2]
```diff
@@ -188,22 +189,26 @@ int main(void)
                     case RASP_TKCTRL_START_CMD:
                     {
                         printf("RASP_TKCTRL_START\r\n");
+                        forward();
```
Comment (Vulbado) [inline rasp_tcp_srv/src/rasp_srv.c:192]: What about backward?

File: rasp_tcp_srv/src/motor_ctrl.c (hunk 1 above)
Comment (vlermuis) [reply rasp_tcp_srv/src/motor_ctrl.c:64]: any suggestion? no more than 100 maybe

File: rasp_tcp_srv/src/rasp_srv.c (hunk 2 above)
Comment (vlermuis) [reply rasp_tcp_srv/src/rasp_srv.c:192]: will add in next commit

**Pull request merged**